src/core/Codend.Contracts/Requests/Project/UpdateProjectRequest.cs

```diff
@@ -7,4 +7,5 @@
 /// <param name="Description">Project description.</param>
 public sealed record UpdateProjectRequest(
     string Name,
-    string? Description);
\ No newline at end of file
+    ShouldUpdateBinder<string?> Description,
+    Guid DefaultStatusId);
```

Comment:
What if I don't want to update default status?

Reply:
You don't have to, it's going to be optional. I've added task to unify all requests and commands cuz it's mess with questionmarks.